src/main/java/CharacterGenerator.java

```diff
@@ -0,0 +1,33 @@
+import java.util.Random;
+
+class CharacterGenerator {
+
+    private char alphaCharArray[] = {'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p',
+            'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z'};
+
+    private char numberCharArray[] = {'0', '1', '2', '3', '4', '5', '6', '7', '8', '9',};
+
+    private char specialCharArray[] = {'£', '$', '!', '#', '@', '+', '-', '/', '*'};
+
+
+    char RandomIndexGenerator(char[] array) {
```

Comment:
Think this name could be better - it's not really generating an index more selecting a random character I think?

Reply:
Method name has now been amended.